src/xdocs/property_types.xml

```diff
@@ -403,6 +403,18 @@
       </ul>
     </section>
 
+      <section name="access_modifiers">
+          <p>This property represents Java access modifiers.</p>
+
+          <ul>
+              <li><code>null</code></li>
```

Comment:
I find adding `null`/`nothing` weird. Users cannot specifically set this value, right?
`Scope` has `nothing` but like I said, they can't specifically write a `nothing` value in the configuration.

Either one should throw an exception that it isn't in the enumeration if users try to use them.
My opinion is we just don't list these. Checks have default values, so that should be enough.